docker-infrastructure.yml

```diff
@@ -34,19 +34,17 @@ services:
     # default 'testclientid' client id
     command: >-
       --client_id=${ACRA_CLIENT_ID:-poison_records_demo}
-      --generate_acraserver_keys
+      --generate_symmetric_storage_key
       --generate_acrawriter_keys
       --keys_output_dir=/keys/acra-server
       --keys_public_output_dir=/keys/acra-server
+      --keystore=v1
 
   # Create poison records:
   acra-poisonrecordmaker:
     # You can specify docker image tag in the environment
     # variable ACRA_DOCKER_IMAGE_TAG or run by default with 'latest' images
-    build:
-      context: .
-      dockerfile: acra/acra-poisonrecordmaker.dockerfile
-    image: "cossacklabs/acra-poisonrecordmaker:${ACRA_DOCKER_IMAGE_TAG:-latest}"
+    image: "cossacklabs/acra-server:${ACRA_DOCKER_IMAGE_TAG:-latest}"
```

Comment:
plus if you touch this demo, can you freeze image's versions and explicitly set 0.92.0?